compiler/codegen/cg-symbol.cpp

```diff
@@ -2903,17 +2892,9 @@ void FnSymbol::codegenDef() {
             //
             if (srcSize > dstSize) {
               irBuilder->CreateMemCpy(ptr,
-#if HAVE_LLVM_VER >= 100
                                       llvm::MaybeAlign(),
-#else
-                                      0,
-#endif
                                       storeAdr,
-#if HAVE_LLVM_VER >= 100
                                       llvm::MaybeAlign(),
-#else
-                                      0,
-#endif
                                       dstSize);
```

Comment:
with all the conditionals gone, can the formatting of this be cleaned up into 1 line?